ocp_resources/cdi_config.py

```diff
@@ -50,3 +50,8 @@ def wait_until_upload_url_changed(self, uploadproxy_url, timeout=TIMEOUT_4MINUTE
                 current_url = status.uploadProxyURL
                 if current_url == uploadproxy_url:
                     return
+
+    @classmethod
+    def is_garbage_collector_enabled_cdi_config(cls):
```

Comment:
Please add docstring that explain that this check `config` CDi and not the current resource that the class represents.

Reply:
ack